zap/src/main/java/org/zaproxy/zap/view/MultipleRegexesOptionsPanel.java

```diff
@@ -89,7 +87,7 @@ public void setRegexes(List<String> regexes) {
     @Override
     public String showAddDialogue() {
         if (addDialog == null) {
-            addDialog = new DialogAddRegex(owner);
+            addDialog = new DialogAddRegex(owner, this.getRegexes());
```

Comment:
It will have to set the list each time (same for modify), it might change after constructing the dialogue.

Reply:
do i have to create new object each time or to create `setRegexes` method in the `DialogAddRegex` class ?

Reply:
I'd go with the setter.

Reply:
The new constructors could be reverted, they are no longer needed.

Reply:
Done @thc202.
Good advice's, I hope doing more development with you in this project.